node/mock/hasherFake.go

```diff
@@ -29,6 +29,6 @@ func (HasherFake) Size() int {
 }
 
 // IsInterfaceNil returns true if there is no value under the interface
-func (sha *HasherFake) IsInterfaceNil() bool {
-	return sha == nil
+func (sha HasherFake) IsInterfaceNil() bool {
```

Comment:
Remove the hasherFake? We are no longer use "fake" in a mock/stub name for a very long time.

Reply:
removed